website/reload_website.sh

```diff
@@ -5,9 +5,11 @@ export PATH="/usr/local/bin:/usr/bin:/bin"
 
 git pull > /tmp/git_changes.txt
 
-if grep -q "Already up to date." /tmp/git_changes.txt; then
+if ! grep -q "gradio/version.txt" /tmp/git_changes.txt; then
```

Comment:
Yeah this runs in a cronjob every minute. 

Yes, `python check_version.py` errors out if it can't find the version on pypi. (if the url `f"https://pypi.org/project/gradio/{VERSION}/"` doesn't return 200) 

In that case the web tracker bot will send a notification to slack that looks like this: 

```
Could not find gradio v3.4.1b10 on pypi: https://pypi.org/project/gradio/3.4.1b10/ does not exist
```

A bit confused though, is this something we do? change version.txt without intending it to upload to pypi?